src/Behat/Gherkin/Filter/LineRangeFilter.php

```diff
@@ -22,14 +22,20 @@
  */
 class LineRangeFilter implements FilterInterface
 {
+    /**
+     * @var int
+     */
     protected $filterMinLine;
+    /**
+     * @var int
+     */
     protected $filterMaxLine;
 
     /**
      * Initializes filter.
      *
-     * @param string $filterMinLine Minimum line of a scenario to filter on
-     * @param string $filterMaxLine Maximum line of a scenario to filter on
+     * @param int|numeric-string $filterMinLine Minimum line of a scenario to filter on
+     * @param int|numeric-string|'*' $filterMaxLine Maximum line of a scenario to filter on
```

Comment:
I verified that PHPStan handles this well: https://phpstan.org/r/bd2527e3-519d-491e-9eb6-7208761748dd